go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/gosimple/slug v1.9.0
 	github.com/grafana/grafana-aws-sdk v0.4.0
 	github.com/grafana/grafana-live-sdk v0.0.6
-	github.com/grafana/grafana-plugin-sdk-go v0.99.0
+	github.com/grafana/grafana-plugin-sdk-go v0.99.1-0.20210524132954-f5c0098028e0
```

Comment:
Can create a tag here

Reply:
Please go ahead tag a new version - v0.100.0 :tada: and update this accordingly. Instructions [here](https://github.com/grafana/grafana-plugin-sdk-go/blob/master/contribute/developer-guide.md#releasing)